tests/phpunit/vfsStreamErroneousFileTestCase.php

```diff
@@ -0,0 +1,303 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of vfsStream.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace bovigo\vfs\tests;
+
+use bovigo\vfs\vfsStream;
+use bovigo\vfs\vfsStreamErroneousFile;
+use const E_USER_WARNING;
+use function bovigo\assert\assertEmptyString;
+use function bovigo\assert\assertFalse;
+use function bovigo\assert\assertThat;
+use function bovigo\assert\assertTrue;
+use function bovigo\assert\expect;
+use function bovigo\assert\predicate\equals;
+use function rand;
+use function uniqid;
+
+/**
+ * Test for bovigo\vfs\vfsStreamErroneousFile.
+ */
+class vfsStreamErroneousFileTestCase extends vfsStreamFileTestCase
+{
+    /**
+     * instance to test
+     *
+     * @var vfsStreamErroneousFile
+     */
+    protected $file;
+
+    /**
+     * set up test environment
+     */
+    protected function setUp(): void
+    {
+        $this->file = vfsStream::newErroneousFile('foo', []);
```

Comment:
`$this->file` is never used throughout the test.

Reply:
It's extending the `vfsStreamFileTestCase` which uses it. This way it tests all the non-error behavior, too.

Reply:
Ah, missed that one.